sql/_13_issues/_24_1h/answers/cbrd_25262.answer

```diff
@@ -0,0 +1,78 @@
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+    
+null     
+
+===================================================
+0
+===================================================
+    
+null     
+
+===================================================
+0
+===================================================
+VIEW    CREATE VIEW    
+public.v1     select _iso88591'public' collate iso88591_bin     
+
+===================================================
+    
+null     
+
+===================================================
+VIEW    CREATE VIEW    
+dba.v1     select _iso88591'dba' collate iso88591_bin     
+
+===================================================
+0
+===================================================
+VIEW    CREATE VIEW    
+dba.v1     select _iso88591'dba' collate iso88591_bin     
+dba.v1     select  cast(_iso88591'dba_1' collate iso88591_bin as varchar(3))     
```

Comment:
There is a different issue than this one.
It must not be cast to `varchar(3)`.
The user's original value is unintentionally changed.